### PR TITLE
Getting blog by ID

### PR DESCRIPTION
**Pull request by PeterKrajci**
Added route for getting blog from DB by ID

### CODE REVIEW

File: src/routes/blogs.js [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+router.get("/", (req, res) => {
+  res.send("We are on blogs");
 });
 
-router.post('/', (req, res) => {
-    console.log(req.body);
+router.get(
+  "/:id",
+  param("id")
+    .isLength({ min: 24, max: 24 })
```
Comment (jakubarendac) [inline src/routes/blogs.js:30]: what is this validation for ? It doesn't make sense from my point of view

File: src/services/blogs.js [hunk 2]
```diff
@@ -1,3 +1,29 @@
-// add code here
+import { validationResult } from "express-validator";
+import { blogModel } from "../models/blog";
 
-export default {}
+const getBlogById = async (req, res) => {
+  try {
+    const validationResults = validationResult(req);
```
Comment (mhalajova) [inline src/services/blogs.js:6]: add new line after this

File: src/services/blogs.js [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+    if (validationResults.isEmpty()) {
+      const idOfBlog = req.params.id;
+      const blog = await blogModel.findById(idOfBlog).exec();
+
+      if (blog == null) {
```
Comment (mhalajova) [inline src/services/blogs.js]: multiple nested if/else statements are difficult to read. Learn not to nest the code unnecessarily

```
if (!validationResults.isEmpty()) {
    // some code
    return;
}

// some code for retrieving a blog
 if (!blog) {
    // some code
    return;
}

res.status(200).send(blog);
```

File: src/models/blog.js [hunk 4]
```diff
@@ -1,46 +1,30 @@
...
-    img:
-    {
-        data: Buffer,
-        contentType: String
-    }
-})
-
-module.exports = mongoose.model('Blogs', blogSchema);
\ No newline at end of file
+export default { blogModel, blogSchema };
```
Comment (mhalajova) [inline src/models/blog.js]: unnecessary export line

File: src/routes/blogs.js (hunk 1 above)
Comment (mhalajova) [inline src/routes/blogs.js:30]: min and max??? isn't it redundant? 
Isnt there function something like `isLength(24)`?

**Review by jakubarendac — CHANGES_REQUESTED** [review]

File: src/routes/blogs.js (hunk 1 above)
Comment (PeterKrajci) [reply src/routes/blogs.js:30]: In this case the id is automatically generated string that contains letters and numbers.  The length of this string id is fixed, so i think that validating input in this way is not totally bad idea.

**Pull request merged**